{% extends 'base.html' %}

{% block title %}
    manual
{% endblock %}
{% block content %}
<div class="container" style="width: 98%;">
<div class="row">
    <div class="col-md-8 col-md-offset-2">

<br>
<h1 class="site__title" style="font-size: 30px;font-weight: 350;">zMAP - Proteomics Analysis Platform</h1>
<ul class="contents" style="list-style-type: none">
    <li>
        <a href="#intro" style="font-weight:bold">1. What is zMAP?</a>
    </li>
    <li>
        <a href="#zMAP" style="font-weight:bold">2. Differential Analysis Module </a>
    </li>
    <ul>
        <li>
            <a href="#zMAP">zMAP</a>
        </li>
        <li>
            <a href="#revese-zMAP">revese-zMAP</a>
        </li>
    </ul>
    <li>
        <a href="#downstream_analysis" style="font-weight:bold">3. Downstream Analysis Module </a>
    </li>
    <ul>
        <li>
            <a href="#QC">Sample quality control</a>
        </li>
        <li>
            <a href="#HVPannotate">HVPs identification and clustering</a>
        </li>
        <li>
            <a href="#Sample_subgroup">Sample subgrouping</a>
        </li>
        <li>
            <a href="#AssClinical">Association with clinical feature</a>
        </li>
        <li>
            <a href="#Surv">Survival analysis</a>
        </li>
        <li>
            <a href="#AssSurv">Association with survival data</a>
        </li>
        <li>
            <a href="#AssMut">Association with mutation data</a>
        </li>
        <li>
            <a href="#GSVA">Gene sets variation analysis</a>
        </li>
        <li>
            <a href="#Network">Co-expression network analysis</a>
        </li>
    </ul>
</ul>


<div class=WordSection1 style='layout-grid:15.6pt'>


<p class=MsoNormal align=left style='margin-left:54.0pt;text-align:left'><span
lang=EN-US>&nbsp;</span></p>

<h1 id="intro" style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>What is zMAP?</span></h1>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>Isobaric
labeling-based mass spectrometry (ILMS) has been widely used to quantify, on a
proteome-wide scale, the relative protein abundance in different biological
conditions. Large-scale proteomic studies based on ILMS, however, typically
involve multiple runs of mass spectrometry (MS), bringing great computational
difficulty to the simultaneous comparison and other integration analyses of all
ILMS samples. We present zMAP, a toolset that makes ILMS intensities comparable
across MS runs by modeling the associated mean-variance dependence and
accordingly applying a variance stabilizing z-transformation. Specific efforts
have been made to render the model fitting procedure resistant to underlying
hypervariable proteins. Two case studies demonstrate the effectiveness of zMAP
in handling large-scale ILMS data sets. Specifically, the transformed
z-statistics, as a new kind of measurements of protein abundance, have
effectively unlocked a broad range of integration analyses that cannot
otherwise be applied on original ILMS intensities.</span></p>

<h1 id="zMAP" style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>zMAP</span></h1>

<p class=aa style='margin-top:23.4pt;margin-right:36.75pt;margin-bottom:23.4pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>The design of zMAP
aims to simultaneously compare protein profiles of multiple samples and
integrate samples from different MS runs for identifying hypervariable proteins
across samples. zMAP models the mean-variance dependence associated with ILMS
intensities and accordingly applies a variance-stabilizing
z-transformation(z-statistic), which dramatically increases the comparability
between samples generated by different MS runs. The z-statistic can be widely
applied in downstream analyses, including PCA, clustering analysis, GSVA, and
so on. To facilitate biologists' usage, we offer a user-friendly and
straightforward web service. zMAP requires that all the involved MS runs are
associated with the same biological design, such that the average intensities across
all samples in each run are biologically identical.</span></p>

<div align=center>

<table style='border-collapse:collapse;border:none'>
 <tr>
  <td width=283 style='width:212.25pt;border:solid #4472C4 1.0pt;border-right:
  none;background:#4472C4;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:white'>Parameter</span></b></p>
  </td>
  <td width=851 style='width:638.25pt;border:solid #4472C4 1.0pt;border-left:
  none;background:#4472C4;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:white'>Description</span></b></p>
  </td>
 </tr>
 <tr>
  <td width=283 style='width:212.25pt;border:solid #8EAADB 1.0pt;border-top:
  none;background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:black'>protein intensity
  file</span></b></p>
  </td>
  <td width=851 style='width:638.25pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US style='color:black'>A tab-delimited file
  containing raw gene-level protein intensity with samples in columns, and gene
  symbols in rows. Note: 1. The protein intensity matrix does not require
  normalization. 2. Sample names can only consist of letters, numbers, and
  underscores.</span></p>
  </td>
 </tr>
 <tr>
  <td width=283 style='width:212.25pt;border:solid #8EAADB 1.0pt;border-top:
  none;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US>sample information file</span></b></p>
  </td>
  <td width=851 style='width:638.25pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US>Sample information file is a three-column,
  tab-delimited file with the <span style='color:black;background:white'>first
  line identifying the columns.</span> <span style='color:black;background:
  white'>The column names are </span></span><span lang=EN-US style='font-family:
  "Courier New";color:black;background:white'>Sample_id,</span><span
  lang=EN-US style='color:black;background:white'> </span><span lang=EN-US
  style='font-family:"Courier New";color:black;background:white'>Sample_condition,</span><span
  lang=EN-US style='color:black;background:white'> and </span><span lang=EN-US
  style='font-family:"Courier New";color:black;background:white'>MS_run</span><span
  lang=EN-US style='color:black;background:white'>.</span></p>
  </td>
 </tr>
 <tr>
  <td width=283 style='width:212.25pt;border:solid #8EAADB 1.0pt;border-top:
  none;background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:black'>window size</span></b></p>
  </td>
  <td width=851 style='width:638.25pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US style='color:black'>The protein number in
  each sliding window, the default is 400.</span></p>
  </td>
 </tr>
 <tr>
  <td width=283 style='width:212.25pt;border:solid #8EAADB 1.0pt;border-top:
  none;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US>step size</span></b></p>
  </td>
  <td width=851 style='width:638.25pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US>The step size for moving the window, the
  default is 100 proteins.</span></p>
  </td>
 </tr>
 <tr>
  <td width=283 style='width:212.25pt;border:solid #8EAADB 1.0pt;border-top:
  none;background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:black'>percent</span></b></p>
  </td>
  <td width=851 style='width:638.25pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US style='color:black'>To avoid the influence
  of differentially expressed proteins across samples, which can lead to an
  overestimated variance, only a certain proportion (30% by default) of the
  proteins with the smallest observed variances are used for the quantile
  regression.</span></p>
  </td>
 </tr>
 <tr>
  <td width=283 style='width:212.25pt;border:solid #8EAADB 1.0pt;border-top:
  none;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US>method</span></b></p>
  </td>
  <td width=851 style='width:638.25pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US>The method used for non-linear fitting, </span><span
  lang=EN-US style='font-family:"Courier New"'>exponential_function </span><span
  lang=EN-US>or </span><span lang=EN-US style='font-family:"Courier New"'>natural_cubic_spline</span></p>
  </td>
 </tr>
</table>

</div>

<h2 style='margin-left:21.0pt'><span lang=EN-US>Input Files</span></h2>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>1. protein intensity file (<a
href="input_file/zMAP/raw_protein_intensity_in_gene_level_for_web.txt">example</a>)</span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>A tab-delimited file
containing raw gene-level protein intensity with samples in columns, and gene
symbols in rows. </span></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><i><span lang=EN-US>Note: </span></i></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:57.75pt;text-indent:30.0pt'><span lang=EN-US>1. The protein
intensity matrix does not require normalization. </span></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:57.75pt;text-indent:30.0pt'><span lang=EN-US>2. Sample names can
only consist of letters, numbers, and underscores.</span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>2. sample information file (<a
href="input_file/zMAP/zmap_sample_info_for_web.txt">example</a>)</span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><a name="_Hlk150620213"><span
lang=EN-US>Sample information file is a three-column, tab-delimited file with
the first line identifying the columns. The column names are Sample_id
Sample_condition and MS_run. </span></a></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><i><span lang=EN-US>Note: </span></i></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:63.0pt;text-indent:30.0pt'><span lang=EN-US style='font-family:
"Courier New"'>Sample_id </span></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:63.0pt;text-indent:60.0pt'><span lang=EN-US style='font-family:
"Courier New"'>Sample_id</span><span lang=EN-US> can only consist of letters,
numbers, and underscores.</span></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:63.0pt;text-indent:30.0pt'><span lang=EN-US style='font-family:
"Courier New"'>Sample_condition </span></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:63.0pt;text-indent:60.0pt'><span lang=EN-US style='font-family:
"Courier New"'>Sample_condition</span><span lang=EN-US> must consist only of
letters, numbers, and underscores.</span></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:63.0pt;text-indent:30.0pt'><span lang=EN-US style='font-family:
"Courier New"'>MS_run</span></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:63.0pt;text-indent:60.0pt'><span lang=EN-US>The MS run in which the
proteins of the sample were quantified.</span></p>

<h2 style='margin-left:21.0pt'><span lang=EN-US>Output Files</span></h2>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>1. Table of z-statistic.</span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US><a
href="input_file/zMAP/zMAP_results/z_statistic_table.txt">z_statistic_table.txt</a>
</span></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>It is crucial for
downstream analysis and visualization.</span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>2. The table containing chi-square statistics and
corresponding statistical p-values.</span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:31.5pt;text-indent:30.0pt'><span lang=EN-US style='color:red'><a
href="input_file/zMAP/zMAP_results/zmap_chi_square_pvalue.txt">zmap_chi_square_pvalue.txt</a></span></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:31.5pt;text-indent:30.0pt'><span lang=EN-US>This file will be
utilized to identify hypervariable proteins across samples.</span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>3. <span class=3>Results directory for each MS run's
model construction.</span></span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:33.0pt;text-indent:30.0pt'><span class=3><span lang=EN-US
style='line-height:150%;font-weight:normal'>The <a
href="input_file/zMAP/zMAP_results">results directory</a> encompasses figures
of normalized data, linear regression, nonlinear regression outcomes, and so
on. Users can consult these resources to evaluate data quality and model
fitting.</span></span></p>

<h1 id="reverse-zMAP" style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>reverse-zMAP</span></h1>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><strong><span lang=EN-US
style='font-family:"Arial",sans-serif;font-weight:normal'> </span></strong></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><strong><span lang=EN-US
style='font-family:"Arial",sans-serif;font-weight:normal'>reverse-zMAP module
is primarily designed for handling MS runs with relatively large sample sizes.
The major concern is that, with the increase of the number of samples, the odds
that outlier measurements are involved for each specific protein increase, giving
rise to excessively large variances. Consequently, in the sliding-window
process of the zMAP module, the proportion of proteins that are suitable to use
for the quantile regression in each window can be very small. Besides, fitting
a single mean-variance curve (MVC) for a large number of samples may not be
flexible enough to allow for the variation of mean-variance trend across
samples. In practice, large-scale proteomic studies have frequently applied the
</span></strong><span class=10><span lang=EN-US style='line-height:150%'>strategy
of adding a biologically identical reference sample to each individual MS run.
For example, in</span></span><strong><span lang=EN-US style='font-family:"Arial",sans-serif;
font-weight:normal'> cancer studies, a mixed sample is typically generated by
pooling tumor samples and/or normal adjacent tissues (NATs) from several
related patients in equal protein amounts. The proteome of this mixed sample is
then profiled in every MS run separately. reverse-zMAP module alleviates the
influence of outliers by repeatedly making pairwise comparisons, which in the
meanwhile allows the modeling of sample-specific mean-variance trend, but it
requires a biologically identical reference sample in each MS run for a
subsequent integration across MS runs.</span></strong></p>

<p class=MsoNormal style='line-height:150%'><span lang=EN-US style='line-height:
150%;font-family:"Arial",sans-serif'>&nbsp;</span></p>

<div align=center>

<table style='border-collapse:collapse;border:none'>
 <tr>
  <td width=274 style='width:205.15pt;border:solid #4472C4 1.0pt;border-right:
  none;background:#4472C4;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><a name="_Hlk148189147"><b><span lang=EN-US
  style='color:white'>Parameter</span></b></a></p>
  </td>
  <td width=860 style='width:645.35pt;border:solid #4472C4 1.0pt;border-left:
  none;background:#4472C4;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:white'>Description</span></b></p>
  </td>
 </tr>
 <tr>
  <td width=274 style='width:205.15pt;border:solid #8EAADB 1.0pt;border-top:
  none;background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:black'>protein intensity
  file</span></b></p>
  </td>
  <td width=860 style='width:645.35pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><a name="_Hlk148117614"><span lang=EN-US style='color:black'>A
  tab-delimited file containing raw gene-level protein intensity with samples
  in columns, and gene symbols in rows. Note: 1. The protein intensity matrix
  does not require normalization. 2. Sample names can only consist of letters,
  numbers, and underscores.</span></a></p>
  </td>
 </tr>
 <tr>
  <td width=274 style='width:205.15pt;border:solid #8EAADB 1.0pt;border-top:
  none;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US>sample information file</span></b></p>
  </td>
  <td width=860 style='width:645.35pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=code0 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span class=a7><span lang=EN-US>The sample information file
  is a four-column, tab-delimited file with the first line identifying the
  columns</span></span><span lang=EN-US>. </span><span class=a7><span
  lang=EN-US>The column names are</span></span><span lang=EN-US> Sample_id,
  MS_run, Sample_condition, </span><span class=a7><span lang=EN-US>and</span></span><span
  lang=EN-US> internal_ref</span><span class=a7><span lang=EN-US>.</span></span><span
  lang=EN-US> </span></p>
  </td>
 </tr>
 <tr>
  <td width=274 style='width:205.15pt;border:solid #8EAADB 1.0pt;border-top:
  none;background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:black'>window size</span></b></p>
  </td>
  <td width=860 style='width:645.35pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US style='color:black'>The protein number in
  each sliding window, the default is 400.</span></p>
  </td>
 </tr>
 <tr>
  <td width=274 style='width:205.15pt;border:solid #8EAADB 1.0pt;border-top:
  none;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US>step size</span></b></p>
  </td>
  <td width=860 style='width:645.35pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US>The step size for moving the window, the
  default is 100 proteins.</span></p>
  </td>
 </tr>
 <tr>
  <td width=274 style='width:205.15pt;border:solid #8EAADB 1.0pt;border-top:
  none;background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:black'>percent</span></b></p>
  </td>
  <td width=860 style='width:645.35pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US style='color:black'>To avoid the influence
  of differential proteins, only a certain proportion (50% by default) of the
  proteins with the middle </span><span
  lang=EN-US style='font-size:10.5pt;font-family:等线;position:relative;
  top:25.5pt'><img width=22 height=83
  src="zMAP_paper_web_tutorial-UTF-8-filtered.files/image001.png"></span><span
  lang=EN-US style='color:black'> are used for quantile regression.</span></p>
  </td>
 </tr>
 <tr>
  <td width=274 style='width:205.15pt;border:solid #8EAADB 1.0pt;border-top:
  none;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US>method</span></b></p>
  </td>
  <td width=860 style='width:645.35pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US>The method used for non-linear fitting, </span><span
  class=code><span lang=EN-US>exponential_function</span></span><span
  class=code><span lang=EN-US style='line-height:150%;font-family:"Arial",sans-serif'>
  </span></span><span lang=EN-US>or </span><span class=code><span lang=EN-US>natural_cubic_spline</span></span><span
  class=code><span lang=EN-US>.</span></span></p>
  </td>
 </tr>
</table>

</div>

<p class=MsoNormal style='line-height:150%'><span lang=EN-US style='font-size:
12.0pt;line-height:150%;font-family:"Arial",sans-serif'>&nbsp;</span></p>

<h2 style='margin-left:21.0pt'><a name="_Hlk148189670"><span lang=EN-US>Input
Files</span></a></h2>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>1. protein intensity file (</span><a
href="input_file/reverse_zMAP/small_data_protein_level_intensity.txt"><span
lang=EN-US>example</span></a><span lang=EN-US>)</span></h3>

<p class=a4 style='margin-top:0cm;margin-right:17.5pt;margin-bottom:7.8pt;
margin-left:31.5pt;text-indent:30.0pt'><span lang=EN-US>A tab-delimited file
containing raw gene-level protein intensity with samples in columns, and gene
symbols in rows. </span></p>

<p class=a4 style='margin-top:0cm;margin-right:17.5pt;margin-bottom:7.8pt;
margin-left:31.5pt;text-indent:30.0pt'><i><span lang=EN-US>Note: </span></i></p>

<p class=a4 style='margin-top:0cm;margin-right:17.5pt;margin-bottom:7.8pt;
margin-left:31.5pt;text-indent:30.0pt'><span lang=EN-US>1. The protein
intensity matrice does not require normalization. </span></p>

<p class=a4 style='margin-top:0cm;margin-right:17.5pt;margin-bottom:7.8pt;
margin-left:31.5pt;text-indent:30.0pt'><span lang=EN-US>2. Sample names can
only consist of letters, numbers, and underscores.</span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>2. sample information file (</span><a
href="input_file/reverse_zMAP/small_data_sample_info_add_internal_ref.txt"><span
lang=EN-US>example</span></a><span lang=EN-US>)</span></h3>

<p class=a4 style='margin-top:0cm;margin-right:17.5pt;margin-bottom:7.8pt;
margin-left:31.5pt;text-indent:30.0pt'><span lang=EN-US>Sample information file
is a four-column, tab-delimited file with the <span style='color:black;
background:white'>first line identifying the columns.</span> <span
style='color:black;background:white'>The column names are </span></span><span
class=code><span lang=EN-US>Sample_id,MS_run,</span></span><span lang=EN-US
style='color:black;background:white'> </span><span class=code><span lang=EN-US>Sample_condition</span></span><span
lang=EN-US style='color:black;background:white'> and </span><span class=code><span
lang=EN-US>internal_ref</span></span><span lang=EN-US style='color:black;
background:white'>. </span></p>

<p class=a4 style='margin-top:0cm;margin-right:17.5pt;margin-bottom:7.8pt;
margin-left:31.5pt;text-indent:30.0pt'><i><span lang=EN-US style='color:black;
background:white'>Note: </span></i></p>

<p class=a4 style='margin-top:0cm;margin-right:17.55pt;margin-bottom:7.8pt;
margin-left:63.0pt;text-indent:30.0pt'><span lang=EN-US style='color:black;
background:white'>To avoid code conflicts,</span><span lang=EN-US> the column
headers are:</span></p>

<p class=code0 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
margin-left:94.5pt'><span lang=EN-US>Sample_id </span></p>

<p class=a4 style='margin-top:0cm;margin-right:17.55pt;margin-bottom:7.8pt;
margin-left:94.5pt;text-indent:30.0pt'><span class=code><span lang=EN-US>Sample_id</span></span><span
lang=EN-US> can only consist of letters, numbers, and underscores.</span></p>

<p class=code0 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
margin-left:94.5pt'><span lang=EN-US>MS_run </span></p>

<p class=a4 style='margin-top:0cm;margin-right:17.55pt;margin-bottom:7.8pt;
margin-left:94.5pt;text-indent:30.0pt'><span lang=EN-US>The MS run in which the
proteins of the sample were quantified.</span></p>

<p class=code0 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
margin-left:94.5pt'><span lang=EN-US>Sample_condition</span></p>

<p class=a4 style='margin-top:0cm;margin-right:17.55pt;margin-bottom:7.8pt;
margin-left:94.5pt;text-indent:30.0pt'><span class=code><span lang=EN-US>Sample_condition</span></span><span
lang=EN-US> <span style='color:black;background:white'>consists only of
letters, numbers, and underscores</span></span><span lang=EN-US
style='font-family:宋体'>.</span></p>

<p class=code0 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
margin-left:94.5pt'><span lang=EN-US>internal_ref </span></p>

<p class=a4 style='margin-top:0cm;margin-right:17.55pt;margin-bottom:7.8pt;
margin-left:94.5pt;text-indent:30.0pt'><span class=code><span lang=EN-US>Yes</span></span><span
lang=EN-US> or </span><span class=code><span lang=EN-US>No</span></span><span
lang=EN-US>.</span><span lang=EN-US> </span><span lang=EN-US>Indicate whether
the sample is a reference sample.</span></p>

<h2 style='margin-left:21.0pt'><span lang=EN-US>Output Files</span></h2>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>1. Table of z-statistic.<span style='color:red'> </span></span></h3>

<p class=a6 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:63.0pt;text-indent:0cm'><a
href="input_file/reverse_zMAP/reverse_zMAP_results/z_statistic_table.txt"><span
lang=EN-US>z_statistic_table.txt</span></a><span lang=EN-US> </span></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:63.0pt;text-indent:0cm'><span lang=EN-US>It is crucial for
downstream analysis and visualization.</span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>2. Chi-square statistics and corresponding statistical
p-values. </span></h3>

<p class=a6 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:45.75pt;text-indent:21.0pt'><a
href="input_file/reverse_zMAP/reverse_zMAP_results/reverse_zmap_chi_square_pvalue.txt"><span
lang=EN-US>reverse_zmap_chi_square_pvalue.txt</span></a></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>This file contains 5
columns: protein, number of samples where this protein was not detected,
chi-square statistic, p-value, BH-corrected p-value, and Bonferroni-corrected
p-value. This file will be utilized to identify hypervariable proteins (HVPs)
across samples.</span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>3. Summary of proteomic datasets at different levels.</span></h3>

<p class=a6 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:42.0pt;text-indent:30.0pt'><a
href="input_file/reverse_zMAP/reverse_zMAP_results/qc_boxplot.pdf"><span
lang=EN-US>qc_boxplot.pdf</span></a><span lang=EN-US> </span></p>

<p class=a6 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:42.0pt;text-indent:30.0pt'><a
href="input_file/reverse_zMAP/reverse_zMAP_results/qc_table.pdf"><span
lang=EN-US>qc_table.pdf</span></a></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>4. Goodness of fit for the application of reverse-zMAP
to your data set.</span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:42.0pt;text-indent:30.0pt'><span lang=EN-US>4.1 Box plot of the R2
values of all quantile regressions. </span></p>

<p class=a6 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:42.0pt;text-indent:60.0pt'><a
href="input_file/reverse_zMAP/reverse_zMAP_results/r2_linear_model_boxplot.pdf"><span
lang=EN-US>r2_linear_model_boxplot.pdf</span></a></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:42.0pt;text-indent:30.0pt'><span lang=EN-US>4.2 The distribution of
R2 values from the fitting of M-A curves. The associated three quartiles are
marked. </span></p>

<p class=a6 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:42.0pt;text-indent:60.0pt'><a
href="input_file/reverse_zMAP/reverse_zMAP_results/r2_distribution_of_nonlinear_model_fitting_for_estimated_variance.pdf.pdf"><span
lang=EN-US>r2_distribution_of_nonlinear_model_fitting_for_estimated_variance.pdf</span></a></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:42.0pt;text-indent:30.0pt'><span lang=EN-US>4.3 The distribution of
R2 values from the fitting of MVCs.</span></p>

<p class=a6 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:42.0pt;text-indent:60.0pt'><a
href="input_file/reverse_zMAP/reverse_zMAP_results/r2_distribution_of_nonlinear_fitting_for_intercept_u.pdf"><span
lang=EN-US>r2_distribution_of_nonlinear_fitting_for_intercept_u.pdf</span></a></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>5. The comparison of cumulative distributions of m
values and z-statistics, with each line representing a sample.</span></h3>

<p class=a6 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:17.45pt;text-indent:54.9pt'><a
href="input_file/reverse_zMAP/reverse_zMAP_results/cumulative_distribution.pdf"><span
lang=EN-US>cumulative_distribution.pdf</span></a></p>

<h1 id="QC" style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>Downstream analysis</span></h1>

<h1 id="QC" style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>Sample quality control </span></h1>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><a name="OLE_LINK10"><span lang=EN-US>Performing
hierarchical clustering and principal component analysis on the z-statistic
matrix of samples provides a concise visualization of the overall impact of
sample conditions and MS runs.</span></a></p>

<div align=center>

<table style='border-collapse:collapse;border:none'>
 <tr>
  <td width=273 style='width:204.75pt;border:solid #4472C4 1.0pt;border-right:
  none;background:#4472C4;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><a name="_Hlk148354246"><b><span lang=EN-US
  style='color:white'>Name</span></b></a></p>
  </td>
  <td width=1315 style='width:985.95pt;border:solid #4472C4 1.0pt;border-left:
  none;background:#4472C4;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:white'>Description</span></b></p>
  </td>
 </tr>
 <tr>
  <td width=273 style='width:204.75pt;border:solid #8EAADB 1.0pt;border-top:
  none;background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:black'>z-statistic file</span></b></p>
  </td>
  <td width=1315 style='width:985.95pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><a name="OLE_LINK5"><span lang=EN-US style='color:black'>Output
  file z_statistic_table.txt from zMAP or reverse-zMAP</span></a></p>
  </td>
 </tr>
 <tr>
  <td width=273 style='width:204.75pt;border:solid #8EAADB 1.0pt;border-top:
  none;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US>sample information file</span></b></p>
  </td>
  <td width=1315 style='width:985.95pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US>The sample information file is a
  three-column, tab-delimited file with the first line identifying the columns.
  The column names are </span><span class=code><span lang=EN-US>Sample_id
  Sample_condition</span></span><span lang=EN-US> and </span><span class=code><span
  lang=EN-US>MS_run</span></span><span lang=EN-US>. Note: To avoid code
  conflicts,</span><span class=code><span lang=EN-US> Sample_condition</span></span><span
  lang=EN-US> consists only of letters, numbers, and underscores.</span></p>
  </td>
 </tr>
</table>

</div>

<h2 style='margin-left:21.0pt'><span lang=EN-US>Input Files</span></h2>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>1. z statistic file (<a
href="input_file/zMAP/zMAP_results/z_statistic_table.txt">example</a>)</span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>Output file <span
class=a5><span style='line-height:150%;color:windowtext'>z_statistic_table.txt</span></span>
from zMAP or reverse-zMAP as input.</span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>2. sample information file (<a
href="input_file/zMAP/zmap_sample_info_for_web.txt">example</a>)</span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>Sample information
file is a three-column, tab-delimited file with the first line identifying the
columns. The column names are </span><span class=code><span lang=EN-US>Sample_id,
Sample_condition,</span></span><span lang=EN-US> and </span><span class=code><span
lang=EN-US>MS_run</span></span><span lang=EN-US>. </span></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><i><span lang=EN-US>Note: </span></i></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>To avoid code
conflicts:</span></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:57.75pt;text-indent:30.0pt'><span lang=EN-US style='font-family:
"Courier New"'>Sample_id</span></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:57.75pt;text-indent:60.0pt'><span lang=EN-US style='font-family:
"Courier New"'>Sample_id</span><span lang=EN-US> must consist of letters,
numbers, and underscores.</span></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:57.75pt;text-indent:30.0pt'><span lang=EN-US style='font-family:
"Courier New"'>Sample_condition</span></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:57.75pt;text-indent:60.0pt'><span lang=EN-US style='font-family:
"Courier New"'>Sample_condition</span><span lang=EN-US> must consist only of
letters, numbers, and underscores.</span></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:57.75pt;text-indent:30.0pt'><span lang=EN-US style='font-family:
"Courier New"'>MS_run</span></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:57.75pt;text-indent:60.0pt'><span lang=EN-US>The MS run in which
the proteins of the sample were quantified.</span></p>

<h2 style='margin-left:21.0pt'><span lang=EN-US>Output Files</span></h2>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span class=3><span lang=EN-US>1. Table of PCC between each pair of
samples based on z-statistic.</span></span></h3>

<p class=a6 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US><a
href="input_file/zMAP/sample_quality_control/pearsonr_correlation_coefficient_of_z_statistic.txt">pearsonr_correlation_coefficient_of_z_statistic.txt</a></span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>2. Sample-based hierarchical cluster.</span></h3>

<p class=a6 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US><a
href="input_file/zMAP/sample_quality_control/pearsonr_correlation_coefficient_of_z_statistic.pdf">pearsonr_correlation_coefficient_of_z_statistic.pdf</a></span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><a name="_Hlk150454825"><span lang=EN-US>3. Principal components
matrix.</span></a></h3>

<p class=a6 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><a
href="input_file/zMAP/sample_quality_control/pca_df.txt"><span lang=EN-US>pca_df.txt</span></a></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>4. Principal component plot.</span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>A principal component
plot displaying samples in a 2D plane defined by their first two principal
components. This plot provides a concise visualization of the overall impact of
sample conditions and MS runs.</span></p>

<p class=a6 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><a
href="input_file/zMAP/sample_quality_control/PC1_PC2_scatterplot.pdf"><span
lang=EN-US>PC1_PC2_scatterplot.pdf</span></a></p>

<h1 id="HVPannotate" style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>HVPs identification and clustering</span></h1>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>The chi-square
statistics derived by zMAP along with the associated numbers of degrees of
freedom were summed across MS runs for each protein, giving rise to a p-value
that assessed the overall expression variability of the protein.</span><span
lang=EN-US> </span><span lang=EN-US>This functionality is designed to select
hypervariable proteins (HVPs) from the output files of zMAP and cluster them to
obtain multiple expression signatures. Then, pathway enrichment analysis was
conducted on these signatures to reveal biological insights.</span></p>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US
style='font-size:12.0pt;font-family:"Arial",sans-serif;color:red'>&nbsp;</span></p>

<div align=center>

<table style='border-collapse:collapse;border:none'>
 <tr>
  <td width=349 style='width:261.7pt;border:solid #4472C4 1.0pt;border-right:
  none;background:#4472C4;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:white'>Parameter</span></b></p>
  </td>
  <td width=785 style='width:588.8pt;border:solid #4472C4 1.0pt;border-left:
  none;background:#4472C4;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:white'>Description</span></b></p>
  </td>
 </tr>
 <tr>
  <td width=349 style='width:261.7pt;border:solid #8EAADB 1.0pt;border-top:
  none;background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:black'>Chi-square and
  p-value file</span></b></p>
  </td>
  <td width=785 style='width:588.8pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US style='color:black'>Output file
  zmap_chi_square_pvalue.txt from zMAP.</span></p>
  </td>
 </tr>
 <tr>
  <td width=349 style='width:261.7pt;border:solid #8EAADB 1.0pt;border-top:
  none;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US>z-statistic file</span></b></p>
  </td>
  <td width=785 style='width:588.8pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US>Output file z_statistic_table.txt from
  zMAP.</span></p>
  </td>
 </tr>
 <tr>
  <td width=349 style='width:261.7pt;border:solid #8EAADB 1.0pt;border-top:
  none;background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:black'>sample information
  file</span></b></p>
  </td>
  <td width=785 style='width:588.8pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US style='color:black'>The sample information
  file is a three-column, tab-delimited file with the first line identifying
  the columns. The column names are </span><span lang=EN-US style='font-family:
  "Courier New";color:black'>Sample_id</span><span lang=EN-US style='font-family:
  "Courier New";color:black'>, </span><span lang=EN-US style='font-family:"Courier New";
  color:black'>Sample_condition</span><span lang=EN-US style='color:black'>,
  and </span><span lang=EN-US style='font-family:"Courier New";color:black'>MS_run</span><span
  lang=EN-US style='color:black'>. </span></p>
  </td>
 </tr>
 <tr>
  <td width=349 style='width:261.7pt;border:solid #8EAADB 1.0pt;border-top:
  none;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US>adjusted p-value cutoff</span></b></p>
  </td>
  <td width=785 style='width:588.8pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US>BH-corrected p-value cutoff for identifying
  HVPs. The default is 0.05.</span></p>
  </td>
 </tr>
 <tr>
  <td width=349 style='width:261.7pt;border:solid #8EAADB 1.0pt;border-top:
  none;background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:black'>number of clusters
  for HVPs </span></b></p>
  </td>
  <td width=785 style='width:588.8pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US style='color:black'>The hierarchical
  clustering of HVPs into multiple clusters reveals diverse expression </span><span
  lang=EN-US style='font-family:等线;color:black'>signature</span><span
  lang=EN-US style='color:black'>s across different sample conditions.</span></p>
  </td>
 </tr>
 <tr>
  <td width=349 style='width:261.7pt;border:solid #8EAADB 1.0pt;border-top:
  none;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US>minimum number of proteins within each
  cluster</span></b></p>
  </td>
  <td width=785 style='width:588.8pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US>If the number of proteins within certain
  clusters falls below the specified minimum count, these clusters will be
  merged into a single cluster labeled as cluster_0. As a result, the final
  number of clusters may be fewer than what the user initially specified.</span></p>
  </td>
 </tr>
 <tr>
  <td width=349 style='width:261.7pt;border:solid #8EAADB 1.0pt;border-top:
  none;background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:black'>number of top-ranked
  DEPs</span></b></p>
  </td>
  <td width=785 style='width:588.8pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US style='color:black'>The z-statistics
  associated with each protein are separately averaged within each sample
  condition. The standard deviation of these average z-statistics for each
  protein is then calculated. Subsequently, proteins are ranked according to
  these standard deviations, and the top-ranked differentially expressed
  proteins (DEP</span><span lang=EN-US style='font-family:等线;color:black'>s</span><span
  lang=EN-US style='color:black'>) are selected.</span></p>
  </td>
 </tr>
 <tr>
  <td width=349 style='width:261.7pt;border:solid #8EAADB 1.0pt;border-top:
  none;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US>number of clusters for top-ranked DEPs</span></b></p>
  </td>
  <td width=785 style='width:588.8pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US>The K-means clustering of top-ranked
  differentially expressed proteins into multiple clusters identifies diverse
  expression patterns across various sample conditions.</span></p>
  </td>
 </tr>
</table>

</div>

<h2 style='margin-left:21.0pt'><span lang=EN-US>Input Files</span></h2>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>1. Chi-square and p-value file (<a
href="input_file/zMAP/zMAP_results/zmap_chi_square_pvalue.txt">example</a>)</span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>Output file
zmap_chi_square_pvalue.txt from zMAP as input.</span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>2. z statistic file (<a
href="input_file/zMAP/zMAP_results/z_statistic_table.txt">example</a>) </span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>Output file<span
class=a5><span style='line-height:150%;color:windowtext'> z_statistic_table.txt</span></span>
from zMAP as input.</span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>3. Sample information file (<a
href="input_file/zMAP/zmap_sample_info_for_web.txt">example</a>)</span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>As above</span></p>

<h2 style='margin-left:21.0pt'><span lang=EN-US>Output Files</span></h2>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>1. Heatmap showing the z-statistics of hypervariable
proteins.</span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>These proteins were
clustered into multiple clusters.</span></p>

<p class=a6 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US><a
href="input_file/zMAP/zmap_hypervariable_proteins_cluster/hypervariable_proteins_cluster_heatmap_bh_cutoff_0.05.pdf">hypervariable_proteins_cluster_heatmap_bh_cutoff_0.05.pdf
</a></span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>2. The list of proteins belonging to each cluster is
within the dictionary.</span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US style='font-family:
"Courier New";color:red'><a
href="input_file/zMAP/zmap_hypervariable_proteins_cluster/clusters">./clusters</a></span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>3. <a name="OLE_LINK12">Pathway enrichment results.</a></span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>Enriched biological
processes and pathways for protein clusters (from GO and keg enrichment
analysis) are integrated within the dictionary.</span></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><a
href="input_file/zMAP/zmap_hypervariable_proteins_cluster/enrichment_results"><span
lang=EN-US style='font-family:"Courier New"'>./enrichment_results</span></a></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>4. Heatmap showing the z-statistics of top-ranked
HVPs.</span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>These proteins were
clustered into multiple clusters.</span></p>

<p class=a6 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US><a
href="input_file/zMAP/zmap_hypervariable_proteins_cluster/top_differentially_expressed_proteins_100/top_100_differentially_expressed_proteins_heatmap.pdf">top_100_differentially_expressed_proteins_heatmap.pdf</a>
</span></p>

<h1 id="Sample_subgroup" style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>Sample subgrouping</span></h1>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>For delineating
molecular subtypes at the protein level, <a name="OLE_LINK8">this function is
designed to select a subset of highly variable proteins across samples</a>. It
then undertakes unsupervised clustering on the samples, offering quantitative
evidence to ascertain both the number and composition of potential clusters
within the dataset. For in-depth algorithmic insights, please consult the
details provided in <i>Consensus Clustering.</i></span><i><span
lang=EN-US><sup>1,2</sup></span></i></p>

<div align=center>

<table style='border-collapse:collapse;border:none'>
 <tr>
  <td width=313 style='width:234.4pt;border:solid #4472C4 1.0pt;border-right:
  none;background:#4472C4;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><a name="_Hlk150518109"></a><a name="_Hlk150525817"><b><span
  lang=EN-US style='color:white'>Name</span></b></a></p>
  </td>
  <td width=1275 style='width:956.3pt;border:solid #4472C4 1.0pt;border-left:
  none;background:#4472C4;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:white'>Description</span></b></p>
  </td>
 </tr>
 <tr>
  <td width=313 style='width:234.4pt;border:solid #8EAADB 1.0pt;border-top:
  none;background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:black'>z-statistic file</span></b></p>
  </td>
  <td width=1275 style='width:956.3pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US style='color:black'>Output file
  z_statistic_table.txt from reverse-zMAP</span></p>
  </td>
 </tr>
 <tr>
  <td width=313 style='width:234.4pt;border:solid #8EAADB 1.0pt;border-top:
  none;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US>sample information file</span></b></p>
  </td>
  <td width=1275 style='width:956.3pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US>The sample information file is a
  three-column, tab-delimited file with the first line identifying the columns.
  The column names are </span><span class=code><span lang=EN-US>Sample_id</span></span><span
  lang=EN-US> </span><span class=code><span lang=EN-US>Sample_condition</span></span><span
  lang=EN-US> and </span><span class=code><span lang=EN-US>MS_run</span></span><span
  lang=EN-US>.</span></p>
  </td>
 </tr>
 <tr>
  <td width=313 style='width:234.4pt;border:solid #8EAADB 1.0pt;border-top:
  none;background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:black'>Sample condition</span></b></p>
  </td>
  <td width=1275 style='width:956.3pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US style='color:black'>perform clustering on
  samples under specific conditions.</span></p>
  </td>
 </tr>
 <tr>
  <td width=313 style='width:234.4pt;border:solid #8EAADB 1.0pt;border-top:
  none;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US>Protein number</span></b></p>
  </td>
  <td width=1275 style='width:956.3pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US>Proteins were ranked based on their
  variance across samples, and the z-statistic matrices of the top_n proteins
  were used for clustering. By default, the top_n is set to 3000.</span></p>
  </td>
 </tr>
</table>

</div>

<h2 style='margin-left:21.0pt'><span lang=EN-US>Input Files</span></h2>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>1.z statistic file (</span><a
href="input_file/reverse_zMAP/cervical_cancer_z_statistic.txt"><span
lang=EN-US>example</span></a><span lang=EN-US>) </span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>Output file
z_statistic_table.txt from reverse-zMAP as input.</span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>2. sample information file (</span><a
href="input_file/reverse_zMAP/cervical_cancer_sample_info.txt"><span
lang=EN-US>example</span></a><span lang=EN-US>) </span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>As above.</span></p>

<h2 style='margin-left:21.0pt'><span lang=EN-US>Output File</span></h2>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>1. Sample clustering results: </span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><a
href="input_file/reverse_zMAP/sample_subgrouping/cluster_3.csv"><span
lang=EN-US>cluster_XX.csv</span></a><span lang=EN-US> <span class=a3>(XX is the
number of clusters)</span></span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>2. Graphic output</span><span lang=EN-US
style='font-family:宋体'>:</span><span lang=EN-US> </span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><a
href="input_file/reverse_zMAP/sample_subgrouping/euclidean_km/consensus.pdf"><span
lang=EN-US>consensus.pdf</span></a></p>

<p class=a4 align=left style='margin-top:0cm;margin-right:36.75pt;margin-bottom:
7.8pt;margin-left:63.0pt;text-align:left;text-indent:3.75pt'><span class=a3><span
lang=EN-US>We invoked the ConsensusClusterPlus R package at the underlying
level. For interpretation of the output graphics, please refer to </span></span><a
href="https://bioconductor.org/packages/release/bioc/vignettes/ConsensusClusterPlus/inst/doc/ConsensusClusterPlus.pdf"><span
lang=EN-US>https://bioconductor.org/packages/release/bioc/vignettes/ConsensusClusterPlus/inst/doc/ConsensusClusterPlus.pdf</span></a><a
name="OLE_LINK3"></a></p>

<p class=a4 align=left style='margin-top:0cm;margin-right:36.75pt;margin-bottom:
7.8pt;margin-left:36.75pt;text-align:left;text-indent:30.0pt'><span lang=EN-US>&nbsp;</span></p>

<h1 id="AssClinical" style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>Association with clinical features</span></h1>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>In the prior step,
samples underwent grouping using the z-statistic matrix derived from HPVs, with
the intent of correlating proteomic subgroups to clinical and molecular
features. We then employed either the Chi-square test or Fisher's exact test to
examine the association between proteomic subgroups and discrete sample
features. This choice was made based on the number of categories within each
feature. For continuous features, we utilized Student’s t-test or ANOVA.</span></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>ANOVA was further
applied to pinpoint differentially expressed proteins (DEPs) among sample
groups. Hierarchical clustering of proteins ensued, driven by the z-statistic
matrix, revealing distinctive expression signatures. Following this, pathway
enrichment analysis was conducted for each set of proteins.</span></p>

<div align=center>

<table style='border-collapse:collapse;border:none'>
 <tr>
  <td width=349 style='width:261.85pt;border:solid #4472C4 1.0pt;border-right:
  none;background:#4472C4;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:white'>Parameter</span></b></p>
  </td>
  <td width=785 style='width:588.65pt;border:solid #4472C4 1.0pt;border-left:
  none;background:#4472C4;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:white'>Description</span></b></p>
  </td>
 </tr>
 <tr>
  <td width=349 style='width:261.85pt;border:solid #8EAADB 1.0pt;border-top:
  none;background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:black'>z-statistic file</span></b></p>
  </td>
  <td width=785 style='width:588.65pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US style='color:black'>Output file
  z_statistic_table.txt from reverse-zMAP.</span></p>
  </td>
 </tr>
 <tr>
  <td width=349 style='width:261.85pt;border:solid #8EAADB 1.0pt;border-top:
  none;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US>cluster file</span></b></p>
  </td>
  <td width=785 style='width:588.65pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US>A comma-separated file with the first
  column containing sample names and the second column indicating the
  respective sample groups.</span></p>
  </td>
 </tr>
 <tr>
  <td width=349 style='width:261.85pt;border:solid #8EAADB 1.0pt;border-top:
  none;background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:black'>Sample clinical and
  molecule file</span></b></p>
  </td>
  <td width=785 style='width:588.65pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><a name="_Hlk150524214"><span lang=EN-US style='color:black'>A
  tab-separated file with the first column representing sample names, and the
  subsequent columns containing clinical and molecular features. Please note
  that column names can only include letters, numbers, and underscores. Allows </span></a><span
  class=code><span lang=EN-US>NaN</span></span><span lang=EN-US
  style='color:black'> values. When plotting, </span><span class=code><span
  lang=EN-US>NaN</span></span><span lang=EN-US style='color:black'> values are
  represented by white color by default.</span></p>
  </td>
 </tr>
 <tr>
  <td width=349 style='width:261.85pt;border:solid #8EAADB 1.0pt;border-top:
  none;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US>discrete features</span></b></p>
  </td>
  <td width=785 style='width:588.65pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US>This is a string where discrete features
  are comma-separated. For example,</span><span class=code><span lang=EN-US>
  'gender, stage'.</span></span></p>
  </td>
 </tr>
 <tr>
  <td width=349 style='width:261.85pt;border:solid #8EAADB 1.0pt;border-top:
  none;background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:black'>Continuous features</span></b></p>
  </td>
  <td width=785 style='width:588.65pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US style='color:black'>This is a string where
  continuous features are comma-separated. For example, </span><span
  class=code><span lang=EN-US>'age, tumor_size'.</span></span></p>
  </td>
 </tr>
 <tr>
  <td width=349 style='width:261.85pt;border:solid #8EAADB 1.0pt;border-top:
  none;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US>color for discrete features</span></b></p>
  </td>
  <td width=785 style='width:588.65pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US>Tab-delimited file with no column names.
  The first column represents discrete sample features, and the second column
  corresponds to the hexadecimal color code for each feature.</span></p>
  </td>
 </tr>
 <tr>
  <td width=349 style='width:261.85pt;border:solid #8EAADB 1.0pt;border-top:
  none;background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:black'>color for continuous
  feature</span></b></p>
  </td>
  <td width=785 style='width:588.65pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US style='color:black'>Tab-delimited file with
  no column names. The first column represents discrete sample features, and
  the second column corresponds to the colormaps in Matplotlib for each
  feature.</span></p>
  </td>
 </tr>
 <tr>
  <td width=349 style='width:261.85pt;border:solid #8EAADB 1.0pt;border-top:
  none;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US>FDR cutoff</span></b></p>
  </td>
  <td width=785 style='width:588.65pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US>FDR cutoff used for identifying DEPs. The
  default FDR is 0.05.</span></p>
  </td>
 </tr>
 <tr>
  <td width=349 style='width:261.85pt;border:solid #8EAADB 1.0pt;border-top:
  none;background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><a name="_Hlk150519421"><b><span lang=EN-US
  style='color:black'>Number of protein clusters</span></b></a></p>
  </td>
  <td width=785 style='width:588.65pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US style='color:black'>Cluster number for
  DEPs.</span></p>
  </td>
 </tr>
</table>

</div>

<h2 style='margin-left:21.0pt'><span lang=EN-US>Input Files</span></h2>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>1.z statistic file (<a
href="input_file/reverse_zMAP/cervical_cancer_z_statistic.txt">example</a>) </span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>Output file
z_statistic_table.txt from reverse-zMAP as input.</span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>2. cluster file (<a
href="input_file/reverse_zMAP/sample_subgrouping/cluster_3.csv">example</a>) </span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>A comma-separated file
with the first column containing sample names and the second column indicating
the respective sample groups.</span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>3. Sample clinical and molecule file (<a
href="input_file/reverse_zMAP/cervical_cancer_clinical_info.txt">example</a>)</span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>A tab-separated file
with the first column representing sample names, and the subsequent columns
containing clinical and molecular features. Please note that column names can
only include letters, numbers, and underscores. Allows </span><span class=code><span
lang=EN-US>NaN</span></span><span lang=EN-US> values. When plotting, </span><span
class=code><span lang=EN-US>NaN </span></span><span lang=EN-US>values are
represented by white color by default.</span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>4. color for discrete features (<a
href="input_file/reverse_zMAP/discrete_feature_color_RGB.txt">example</a>)</span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>Tab-delimited file
with no column names. The first column represents discrete sample features, and
the second column corresponds to the <a name="OLE_LINK9">hexadecimal color code</a>
(such as </span><span class=code><span lang=EN-US>#808080</span></span><span
lang=EN-US>) for each feature.</span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>5. color for continuous feature (<a
href="input_file/reverse_zMAP/continuous_feature_colorbar.txt">example</a>)</span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>Tab-delimited file
with no column names. The first column represents discrete sample features, and
the second column corresponds to the colormaps in <i>Matplotlib</i> for each
feature. </span></p>

<p class=MsoNormal style='margin-left:18.0pt;text-indent:45.0pt;line-height:
150%'><span lang=EN-US style='font-size:15.0pt;line-height:150%;font-family:
"Arial",sans-serif'>Colormaps: </span><span lang=EN-US style='font-size:15.0pt;
line-height:150%'><a
href="https://matplotlib.org/stable/users/explain/colors/colormaps.html"><span
style='font-family:"Arial",sans-serif'>https://matplotlib.org/stable/users/explain/colors/colormaps.html</span></a></span></p>

<h2 style='margin-left:21.0pt'><span lang=EN-US>Output File</span></h2>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>1. Graphic output: </span></h3>

<p class=a6 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US><a
href="input_file/reverse_zMAP/association_with_clinical_and_molecule_feature/sample_clustering_association_with_clinical_and_molecule_feature.pdf">sample_clustering_association_with_clinical_and_molecule_feature.pdf</a></span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>2. pathway enrichment results:</span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span class=a3><span lang=EN-US>The
directory contains pathway enrichment results for each expression signature.</span></span></p>

<p class=code0 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
margin-left:36.75pt;text-indent:26.25pt'><span lang=EN-US><a
href="input_file/reverse_zMAP/association_with_clinical_and_molecule_feature/enrichment_results"><span
style='line-height:150%'>./</span><b><span style='line-height:150%;font-family:
"Arial",sans-serif'>enrichment_results</span></b></a></span></p>

<h1 id="Surv" style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>Survival analysis</span></h1>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>Plots the Kaplan-Meier
survival curve and assesses the significance of survival differences among
subgroups.</span></p>

<div align=center>

<table style='border-collapse:collapse;border:none'>
 <tr>
  <td width=406 style='width:304.55pt;border:solid #4472C4 1.0pt;border-right:
  none;background:#4472C4;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:white'>Parameter</span></b></p>
  </td>
  <td width=802 style='width:601.4pt;border:solid #4472C4 1.0pt;border-left:
  none;background:#4472C4;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:white'>Description</span></b></p>
  </td>
 </tr>
 <tr>
  <td width=406 style='width:304.55pt;border:solid #8EAADB 1.0pt;border-top:
  none;background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:black'>survival and group
  file</span></b></p>
  </td>
  <td width=802 style='width:601.4pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US style='color:black'>The sample information
  file is a four-column, tab-delimited file with the first line identifying the
  columns. The column names are </span><span class=code><span lang=EN-US>Sample_id,
  survival_time,</span></span><span lang=EN-US style='color:black'> </span><span
  class=code><span lang=EN-US>death_or_not,</span></span><span lang=EN-US
  style='color:black'> and </span><span class=code><span lang=EN-US>group</span></span><span
  lang=EN-US style='color:black'>.</span></p>
  </td>
 </tr>
</table>

</div>

<h2 style='margin-left:21.0pt'><a name="_Hlk150539677"><span lang=EN-US>Input
Files</span></a></h2>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>1. survival and group file (</span><a
href="input_file/reverse_zMAP/survival_analysis_data_for_web.txt"><span
lang=EN-US>example</span></a><span lang=EN-US>) </span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>Sample information
file is a four-column, tab-delimited file with the first line identifying the
columns. The column names are </span><span class=code><span lang=EN-US>Sample_id,
survival_time, death_or_not</span></span><span lang=EN-US> and </span><span
class=code><span lang=EN-US>group</span></span><span lang=EN-US>.</span></p>

<h2 style='margin-left:21.0pt'><span lang=EN-US>Output File</span></h2>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>1. Graphic output: </span></h3>

<p class=a6 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><a
href="input_file/reverse_zMAP/survival_analysis/survival_analysis.pdf"><span
lang=EN-US>survival_analysis.pdf</span></a></p>

<h1 id="AssSurv" style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>Association with survival data</span></h1>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>Cox proportional
hazard regression model is used to identify prognostic markers based on
z-statistic.</span><span
lang=EN-US><sup>3</sup></span></p>

<div align=center>

<table style='border-collapse:collapse;border:none'>
 <tr>
  <td width=381 style='width:285.95pt;border:solid #4472C4 1.0pt;border-right:
  none;background:#4472C4;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><a name="_Hlk150542632"><b><span lang=EN-US
  style='color:white'>Parameter</span></b></a></p>
  </td>
  <td width=753 style='width:564.55pt;border:solid #4472C4 1.0pt;border-left:
  none;background:#4472C4;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:white'>Description</span></b></p>
  </td>
 </tr>
 <tr>
  <td width=381 style='width:285.95pt;border:solid #8EAADB 1.0pt;border-top:
  none;background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:black'>z-statistic file</span></b></p>
  </td>
  <td width=753 style='width:564.55pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US style='color:black'>Output file z_statistic_table.txt
  from reverse-zMAP</span></p>
  </td>
 </tr>
 <tr>
  <td width=381 style='width:285.95pt;border:solid #8EAADB 1.0pt;border-top:
  none;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><a name="_Hlk150537419"><b><span lang=EN-US>survival file</span></b></a></p>
  </td>
  <td width=753 style='width:564.55pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US>The sample information file is a
  three-column, tab-delimited file with the first line identifying the columns.
  The column names are </span><span class=code><span lang=EN-US>Sample_id,
  survival_time,</span></span><span lang=EN-US> and </span><span class=code><span
  lang=EN-US>death_or_not</span></span><span lang=EN-US>. </span></p>
  </td>
 </tr>
</table>

</div>

<h2 style='margin-left:21.0pt'><span lang=EN-US>Input Files</span></h2>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>1. z-statistic file (<a
href="input_file/reverse_zMAP/HCC_z_statistic_df.txt">example</a>) </span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>Output file
z_statistic_table.txt from reverse-zMAP.</span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>2. survival file (<a
href="input_file/reverse_zMAP/survival_analysis_data_for_web.txt">example</a>)</span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>Sample information
file is a three-column, tab-delimited file with the first line identifying the
columns. The column names are </span><span class=code><span lang=EN-US>Sample_id,
survival_time,</span></span><span lang=EN-US> and</span><span class=code><span
lang=EN-US> death_or_not</span></span><span lang=EN-US>.</span></p>

<h2 style='margin-left:21.0pt'><span lang=EN-US>Output File</span></h2>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>1. Cox-regression results file.</span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>Proteins are
categorized into three groups: favorable, unfavorable, and not prognostic.</span></p>

<p class=a6 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US><a
href="input_file/reverse_zMAP/association_z_statistic_with_survival/results_cox_regression.txt">results_cox_regression.txt</a></span></p>

<h1 id="AssMut" style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>Association with mutation data</span></h1>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span class=a3><span lang=EN-US>To pick
out the proteins that were associated with non-silent somatic mutations, QTL
analysis is used.</span></span><span lang=EN-US> Somatic mutations in at least
10 samples (Users determine based on the number of samples.). Then, we pick out
the proteins whose expression was detected in at least half of the tumor
samples. Finally, for each candidate gene-protein pair, we performed a
regression of the protein expression against the mutation status of the gene
using </span><span class=code><span lang=EN-US>MatrixEQTL</span></span><span
lang=EN-US> R package</span><span
lang=EN-US><sup>4</sup></span><span lang=EN-US>, by fitting the following
linear model:</span></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span
lang=EN-US style='font-size:10.5pt;font-family:等线'><img width=230 height=52
src="zMAP_paper_web_tutorial-UTF-8-filtered.files/image002.png"></span></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span
lang=EN-US style='font-size:10.5pt;font-family:等线'><img width=116 height=52
src="zMAP_paper_web_tutorial-UTF-8-filtered.files/image003.png"></span></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>Here, </span><span
lang=EN-US style='font-size:10.5pt;font-family:等线;position:relative;top:17.5pt'><img
width=13 height=52
src="zMAP_paper_web_tutorial-UTF-8-filtered.files/image004.png"></span><span
lang=EN-US> was a vector of the z-statistics (derived by reverse-zMAP) of the
protein in tumor samples; </span><span
lang=EN-US style='font-size:10.5pt;font-family:等线;position:relative;top:17.5pt'><img
width=13 height=52
src="zMAP_paper_web_tutorial-UTF-8-filtered.files/image005.png"></span><span
lang=EN-US> was the non-silent somatic mutation indicators of the gene; </span><span
lang=EN-US style='font-size:10.5pt;font-family:等线;position:relative;top:17.5pt'><img
width=12 height=52
src="zMAP_paper_web_tutorial-UTF-8-filtered.files/image006.png"></span><span
lang=EN-US> and </span><span lang=EN-US style='font-size:10.5pt;font-family:
等线;position:relative;top:17.5pt'><img width=13 height=52
src="zMAP_paper_web_tutorial-UTF-8-filtered.files/image007.png"></span><span
lang=EN-US> referred to the sex and age respectively</span><span lang=EN-US
style='font-family:宋体'>,</span><span lang=EN-US> as well as other control
variable; </span><span lang=EN-US style='font-size:10.5pt;font-family:等线;
position:relative;top:17.5pt'><img width=10 height=52
src="zMAP_paper_web_tutorial-UTF-8-filtered.files/image008.png"></span><span
lang=EN-US> was a vector of independent and identically distributed noise
variables; </span><span lang=EN-US>, </span><span
lang=EN-US style='font-size:10.5pt;font-family:等线;position:relative;top:17.5pt'><img
width=13 height=52
src="zMAP_paper_web_tutorial-UTF-8-filtered.files/image009.png"></span><span
lang=EN-US>, </span><span lang=EN-US style='font-size:10.5pt;font-family:等线;
position:relative;top:17.5pt'><img width=12 height=52
src="zMAP_paper_web_tutorial-UTF-8-filtered.files/image010.png"></span><span
lang=EN-US>, and </span><span lang=EN-US style='font-size:10.5pt;font-family:
等线;position:relative;top:17.5pt'><img width=12 height=52
src="zMAP_paper_web_tutorial-UTF-8-filtered.files/image011.png"></span><span
lang=EN-US> were unknown parameters. This model was fitted by applying the
least squares method, and the null hypothesis </span><span
lang=EN-US style='font-size:10.5pt;font-family:等线;position:relative;top:17.5pt'><img
width=50 height=52
src="zMAP_paper_web_tutorial-UTF-8-filtered.files/image012.png"></span><span
lang=EN-US> was then tested by applying the <i>t</i>-test.</span></p>

<div align=center>

<table style='border-collapse:collapse;border:none'>
 <tr>
  <td width=394 style='width:295.85pt;border:solid #4472C4 1.0pt;border-right:
  none;background:#4472C4;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:white'>Parameter</span></b></p>
  </td>
  <td width=740 style='width:554.65pt;border:solid #4472C4 1.0pt;border-left:
  none;background:#4472C4;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:white'>Description</span></b></p>
  </td>
 </tr>
 <tr>
  <td width=394 style='width:295.85pt;border:solid #8EAADB 1.0pt;border-top:
  none;background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><a name="_Hlk150542679"><b><span lang=EN-US
  style='color:black'>z-statistic fil</span></b></a></p>
  </td>
  <td width=740 style='width:554.65pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US style='color:black'>Output file
  z_statistic_table.txt from reverse-zMAP</span></p>
  </td>
 </tr>
 <tr>
  <td width=394 style='width:295.85pt;border:solid #8EAADB 1.0pt;border-top:
  none;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US>mutation file</span></b></p>
  </td>
  <td width=740 style='width:554.65pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US>Tab-delimited file where rows represent
  genes, columns represent samples, and values are coded as 1 for non-silent
  mutations and 0 for no non-silent mutations.</span></p>
  </td>
 </tr>
 <tr>
  <td width=394 style='width:295.85pt;border:solid #8EAADB 1.0pt;border-top:
  none;background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><a name="_Hlk150545985"><b><span lang=EN-US
  style='color:black'>covariates file</span></b></a></p>
  </td>
  <td width=740 style='width:554.65pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US style='color:black'>The file includes
  additional covariates. Columns represent samples, while rows encompass
  various additional covariates, including age, sex, and others. The values
  inside must be numerical.</span></p>
  </td>
 </tr>
 <tr>
  <td width=394 style='width:295.85pt;border:solid #8EAADB 1.0pt;border-top:
  none;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US>gene position file</span></b></p>
  </td>
  <td width=740 style='width:554.65pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US>The file containing transcription start
  site information for all genes in the genome. It contains three columns with
  the column names being </span><span class=code><span lang=EN-US>gene,
  chromosome,</span></span><span lang=EN-US> and </span><span class=code><span
  lang=EN-US>tss</span></span><span lang=EN-US> respectively.</span></p>
  </td>
 </tr>
 <tr>
  <td width=394 style='width:295.85pt;border:solid #8EAADB 1.0pt;border-top:
  none;background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:black'>chromosome length
  file</span></b></p>
  </td>
  <td width=740 style='width:554.65pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US style='color:black'>Tab-delimited file
  contains two columns, </span><span class=code><span lang=EN-US>chromosome</span></span><span
  lang=EN-US style='color:black'> and </span><span class=code><span lang=EN-US>length</span></span><span
  lang=EN-US style='color:black'>.</span></p>
  </td>
 </tr>
 <tr>
  <td width=394 style='width:295.85pt;border:solid #8EAADB 1.0pt;border-top:
  none;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US>FDR cutoff</span></b></p>
  </td>
  <td width=740 style='width:554.65pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US>The default FDR is 0.05.</span></p>
  </td>
 </tr>
</table>

</div>

<h2 style='margin-left:21.0pt'><span lang=EN-US>Input Files</span></h2>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>1. z-statistic file (<a
href="input_file/reverse_zMAP/HCC_z_statistic_df.txt">example</a>) </span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>Output file <span
class=a5><span style='line-height:150%;color:windowtext'>z_statistic_table.txt</span></span>
from reverse-zMAP.</span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>2. mutation file (<a
href="input_file/reverse_zMAP/snp_df_10.txt">example</a>)</span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>Tab-delimited file
where rows represent genes, columns represent samples, and values are coded as
1 for non-silent mutations and 0 for no non-silent mutations.</span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>3. covariates file (<a
href="input_file/reverse_zMAP/covariates_df.txt">example</a>)</span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>The file includes
additional covariates. Columns represent samples, while rows encompass various
additional covariates, including age, sex, and others. The values inside must
be numerical.</span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>4. gene position file (<a
href="input_file/reverse_zMAP/gene_tss_location.txt">example</a>)</span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>The file containing
transcription start site information for all genes in the genome. It contains
three columns with the column names being </span><span class=code><span
lang=EN-US>gene, chromosome,</span></span><span lang=EN-US> and </span><span
class=code><span lang=EN-US>tss</span></span><span lang=EN-US> respectively.</span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>5. chromosome length file (<a
href="input_file/reverse_zMAP/hg19_chrom_sizes_remove_xy.txt">example</a>)</span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>Tab-delimited file
contains two columns, </span><span class=code><span lang=EN-US>chromosome </span></span><span
lang=EN-US>and </span><span class=code><span lang=EN-US>length</span></span><span
lang=EN-US>.</span></p>

<h2 style='margin-left:21.0pt'><span lang=EN-US>Output File</span></h2>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>1. Graphic output:</span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span class=a5><span lang=EN-US
style='line-height:150%;color:windowtext'><a
href="input_file/reverse_zMAP/association_z_statistic_with_mutation_data/pQTL_results_FDR_0.05.pdf">pQTL_results_FDR_XX.pdf</a></span></span><span
lang=EN-US> (XX is FDR cutoff)</span></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>Two-dimensional plot
displaying the significant gene-protein associations, with Y and X axes
representing the locations of proteins and mutated genes in the genome,
respectively. The total number of proteins associated with each mutated gene is
also displayed. For a specific gene-protein association, Beta&gt;0 suggests the
protein has increased expression with the mutation of the gene, and vice versa.</span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>2. proteins associated with mutation genes. </span></h3>

<p class=a6 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US><a
href="input_file/reverse_zMAP/association_z_statistic_with_mutation_data/results_association_z_statistic_with_mutation.txt">results_association_z_statistic_with_mutation.txt</a></span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>3. Number of proteins associated with mutated genes.</span></h3>

<p class=a6 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US><a
href="input_file/reverse_zMAP/association_z_statistic_with_mutation_data/mutation_associated_protein_count.txt">mutation_associated_protein_count.txt</a></span></p>

<p class=a6 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>&nbsp;</span></p>

<h1 id="GSVA" style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>Gene set variation analysis</span></h1>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><a name="_Hlk150795351"><span
lang=EN-US>GSVA calculates gene set enrichment scores (GSVA scores) for each
sample</span></a><span
lang=EN-US><sup>5</sup></span><span lang=EN-US> using the z-statistic matrix.
Differential expression analysis is then conducted on these GSVA scores using
limma</span><span
lang=EN-US><sup>6</sup></span><span lang=EN-US>, aiming to identify changes in
pathway activities across multiple sample groups. Finally, the differential
pathway activities across sample groups are visualized using a heatmap.</span></p>

<p class=MsoNormal style='line-height:150%'><span lang=EN-US style='font-size:
12.0pt;line-height:150%;font-family:"Arial",sans-serif'>&nbsp;</span></p>

<div align=center>

<table style='border-collapse:collapse;border:none'>
 <tr>
  <td width=141 style='width:106.1pt;border:solid #4472C4 1.0pt;border-right:
  none;background:#4472C4;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:white'>Parameter</span></b></p>
  </td>
  <td width=279 style='width:209.5pt;border:solid #4472C4 1.0pt;border-left:
  none;background:#4472C4;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:white'>Description</span></b></p>
  </td>
 </tr>
 <tr>
  <td width=141 style='width:106.1pt;border:solid #8EAADB 1.0pt;border-top:
  none;background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US style='color:black'>z-statistic file</span></b></p>
  </td>
  <td width=279 style='width:209.5pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US style='color:black'>Output file
  z_statistic_table.txt from zMAP or reverse-zMAP as input.</span></p>
  </td>
 </tr>
 <tr>
  <td width=141 style='width:106.1pt;border:solid #8EAADB 1.0pt;border-top:
  none;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US>sample information file</span></b></p>
  </td>
  <td width=279 style='width:209.5pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US>Sample information file is a three-column,
  tab-delimited file with the <span style='color:#333333;background:white'>first
  line identifying the columns.</span> <span style='color:#333333;background:
  white'>The column names are </span></span><span class=code><span lang=EN-US>Sample_id,
  Sample_condition</span></span><span lang=EN-US style='color:#333333;
  background:white'> and </span><span class=code><span lang=EN-US>MS_run.</span></span><span
  lang=EN-US style='color:#333333;background:white'> </span></p>
  </td>
 </tr>
 <tr>
  <td width=141 style='width:106.1pt;border:solid #8EAADB 1.0pt;border-top:
  none;background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><a name="OLE_LINK16"><b><span lang=EN-US style='color:black'>number
  of top-ranked differentially active gene sets</span></b></a></p>
  </td>
  <td width=279 style='width:209.5pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US style='color:black'>Extract a table of the
  top-ranked differentially active biological processes and KEGG pathways
  across sample groups.</span></p>
  </td>
 </tr>
 <tr>
  <td width=141 style='width:106.1pt;border:solid #8EAADB 1.0pt;border-top:
  none;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><b><span lang=EN-US>Adjusted pvalue cutoff</span></b></p>
  </td>
  <td width=279 style='width:209.5pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=a8 style='margin-top:15.6pt;margin-right:0cm;margin-bottom:15.6pt;
  margin-left:0cm'><span lang=EN-US>Adjusted pvalue cutoff for the
  differentially active. biological processes and pathways across sample
  groups. The default is 0.05.</span></p>
  </td>
 </tr>
</table>

</div>

<h2 style='margin-left:21.0pt'><span lang=EN-US>Input Files</span></h2>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>1.z statistic file (example) </span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>Output file <span
style='color:red'>z_statistic_table.txt</span> from zMAP or reverse-zMAP as
input.</span></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>2.sample information file (example) </span></h3>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>As above</span></p>

<h2 style='margin-left:21.0pt'><span lang=EN-US>Output Files</span></h2>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>1. <a name="OLE_LINK18">Tables displaying GSVA scores
with samples in columns and gene sets in rows</a></span></h3>

<p class=a6 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><a
href="input_file/zMAP/gsva/kegg_gsva_kcdf_Gaussian.csv"><span lang=EN-US>kegg_gsva_kcdf_Gaussian.csv</span></a></p>

<p class=a6 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><a
href="input_file/zMAP/gsva/go_gsva_kcdf_Gaussian.csv"><span lang=EN-US>go_gsva_kcdf_Gaussian.csv</span></a></p>

<h3 style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>2. Table listing the top-ranked differentially active
biological processes and pathways across sample groups</span></h3>

<p class=a6 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><a
href="input_file/zMAP/gsva/kegg_gsva_differential_pathway_activity_top_50.csv"><span
lang=EN-US>kegg_gsva_differential_pathway_activity_top_XX.csv</span></a><span
lang=EN-US> </span><span class=a3><span lang=EN-US style='color:black'>(XX is
the number of top-ranked differentially active gene sets)</span></span></p>

<p class=a6 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><a
href="input_file/zMAP/gsva/go_gsva_differential_pathway_activity_top_50.csv"><span
lang=EN-US>go_gsva_differential_pathway_activity_top_XX.csv</span></a><span
lang=EN-US> </span><span lang=EN-US style='color:black'>(XX is the number of
top-ranked differentially active gene sets)</span></p>

<p class=MsoTitle style='text-align:justify;text-justify:inter-ideograph;
text-indent:22.1pt'><span lang=EN-US style='font-size:22.0pt;font-family:"Arial",sans-serif'>3.
Heatmap of differential pathway activities:</span></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><a
href="input_file/zMAP/gsva/kegg_gsva_differential_pathway_activity_top_50_adj.P.Val_0.05.pdf"><span
lang=EN-US>kegg_gsva_differential_pathway_activity_top_XX_adj.P.Val_XXX.pdf</span></a></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><a
href="input_file/zMAP/gsva/go_gsva_differential_pathway_activity_top_50_adj.P.Val_0.05.pdf"><span
lang=EN-US>go_gsva_differential_pathway_activity_top_XX_adj.P.Val_XXX.pdf </span></a></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>(both PDF and PNG, XX
is the number of top-ranked differentially active gene sets</span><span
class=a3><span style='font-family:宋体'>，</span></span><span class=a3><span
lang=EN-US>XXX is the adjusted P-value cutoff for differential activity pathway</span></span><span
lang=EN-US>)</span></p>

<h1 id="Network" style='margin-top:23.4pt;margin-right:0cm;margin-bottom:23.4pt;margin-left:
21.0pt'><span lang=EN-US>Co-expression network</span></h1>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>A protein
co-expression network is constructed based on the z-statistics</span><span
lang=EN-US style='font-family:宋体'>.</span><span lang=EN-US> Proteins detected
in more than half of the samples are included in the analysis. We first used
the KNN method to impute missing values in the z-statistic matrix. Then, we
obtained the PTCCs for all pairs of the proteins by applying a previously
developed method for covariance matrix estimation</span><span
lang=EN-US><sup>7</sup></span><span lang=EN-US>. Finally, the statistical
significance of each PTCC was assessed based on a mixture model</span><span
lang=EN-US><sup>8</sup></span><span lang=EN-US>. When constructing the
co-expression network, an edge was added to link a protein pair if and only if
the corresponding PTCC was positive and significant.</span></p>

<div align=center>

<table style='border-collapse:collapse;border:none'>
 <tr>
  <td width=381 style='width:285.95pt;border:solid #4472C4 1.0pt;border-right:
  none;background:#4472C4;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=MsoNormal style='margin-top:15.6pt;margin-right:0cm;margin-bottom:
  15.6pt;margin-left:0cm;line-height:150%'><b><span lang=EN-US
  style='font-size:15.0pt;line-height:150%;font-family:"Arial",sans-serif;
  color:white'>Parameter</span></b></p>
  </td>
  <td width=753 style='width:564.55pt;border:solid #4472C4 1.0pt;border-left:
  none;background:#4472C4;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=MsoNormal style='margin-top:15.6pt;margin-right:0cm;margin-bottom:
  15.6pt;margin-left:0cm;line-height:150%'><b><span lang=EN-US
  style='font-size:15.0pt;line-height:150%;font-family:"Arial",sans-serif;
  color:white'>Description</span></b></p>
  </td>
 </tr>
 <tr>
  <td width=381 style='width:285.95pt;border:solid #8EAADB 1.0pt;border-top:
  none;background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=MsoNormal style='margin-top:15.6pt;margin-right:0cm;margin-bottom:
  15.6pt;margin-left:0cm;line-height:150%'><b><span lang=EN-US
  style='font-size:15.0pt;line-height:150%;font-family:"Arial",sans-serif;
  color:black'>z-statistic file</span></b></p>
  </td>
  <td width=753 style='width:564.55pt;border-top:none;border-left:none;
  border-bottom:solid #8EAADB 1.0pt;border-right:solid #8EAADB 1.0pt;
  background:#D9E2F3;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=MsoNormal style='margin-top:15.6pt;margin-right:0cm;margin-bottom:
  15.6pt;margin-left:0cm;line-height:150%'><span lang=EN-US style='font-size:
  15.0pt;line-height:150%;font-family:"Arial",sans-serif;color:black'>Output
  file z_statistic_table.txt from zMAP or reverse-zMAP as input.</span></p>
  </td>
 </tr>
</table>

</div>

<p class=MsoNormal style='margin-top:18.0pt;margin-right:0cm;margin-bottom:
18.0pt;margin-left:21.0pt;line-height:170%;page-break-after:avoid'><b><span
lang=EN-US style='font-size:26.0pt;line-height:170%;font-family:"Arial",sans-serif'>Input
Files</span></b></p>

<p class=MsoNormal style='margin-top:23.4pt;margin-right:0cm;margin-bottom:
23.4pt;margin-left:21.0pt;line-height:150%;page-break-after:avoid'><b><span
lang=EN-US style='font-size:22.0pt;line-height:150%;font-family:"Arial",sans-serif'>1.z
statistic file (example) </span></b></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US>Output file
z_statistic_table.txt from zMAP or reverse-zMAP as input.</span></p>

<p class=MsoNormal style='margin-top:18.0pt;margin-right:0cm;margin-bottom:
18.0pt;margin-left:21.0pt;line-height:170%;page-break-after:avoid'><b><span
lang=EN-US style='font-size:26.0pt;line-height:170%;font-family:"Arial",sans-serif'>Output
Files</span></b></p>

<p class=MsoNormal style='margin-top:23.4pt;margin-right:0cm;margin-bottom:
23.4pt;margin-left:21.0pt;line-height:150%;page-break-after:avoid'><b><span
lang=EN-US style='font-size:22.0pt;line-height:150%;font-family:"Arial",sans-serif'>1.
protein-protein interaction pairs</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='margin-top:23.4pt;margin-right:0cm;margin-bottom:
23.4pt;margin-left:21.0pt;line-height:150%;page-break-after:avoid'><b><span
lang=EN-US style='font-size:22.0pt;line-height:150%;font-family:"Arial",sans-serif'>Reference</span></b></p>

<p class=EndNoteBibliography0 style='margin-left:36.0pt;text-indent:-36.0pt'><span lang=EN-US>1          Monti, S., Tamayo, P., Mesirov, J. &amp; Golub, T.
Consensus clustering: A resampling-based method for class discovery and
visualization of gene expression microarray data. <i>Mach Learn</i> <b>52</b>,
91-118, doi:Doi 10.1023/A:1023949509487 (2003).</span></p>

<p class=EndNoteBibliography0 style='margin-left:36.0pt;text-indent:-36.0pt'><span
lang=EN-US style='font-family:"Arial",sans-serif'>2          Wilkerson, M. D.
&amp; Hayes, D. N. ConsensusClusterPlus: a class discovery tool with confidence
assessments and item tracking. <i>Bioinformatics</i> <b>26</b>, 1572-1573,
doi:10.1093/bioinformatics/btq170 (2010).</span></p>

<p class=EndNoteBibliography0 style='margin-left:36.0pt;text-indent:-36.0pt'><span
lang=EN-US style='font-family:"Arial",sans-serif'>3          Cox, D. R.
Regression models and life</span><span style='font-family:宋体'>‐</span><span
lang=EN-US style='font-family:"Arial",sans-serif'>tables. <i>Journal of the
Royal Statistical Society: Series B (Methodological)</i> <b>34</b>, 187-202
(1972).</span></p>

<p class=EndNoteBibliography0 style='margin-left:36.0pt;text-indent:-36.0pt'><span
lang=EN-US style='font-family:"Arial",sans-serif'>4          Shabalin, A. A.
Matrix eQTL: ultra fast eQTL analysis via large matrix operations. <i>Bioinformatics</i>
<b>28</b>, 1353-1358 (2012).</span></p>

<p class=EndNoteBibliography0 style='margin-left:36.0pt;text-indent:-36.0pt'><span
lang=EN-US style='font-family:"Arial",sans-serif'>5          Hänzelmann, S.,
Castelo, R. &amp; Guinney, J. GSVA: gene set variation analysis for microarray
and RNA-seq data. <i>BMC bioinformatics</i> <b>14</b>, 1-15 (2013).</span></p>

<p class=EndNoteBibliography0 style='margin-left:36.0pt;text-indent:-36.0pt'><span
lang=EN-US style='font-family:"Arial",sans-serif'>6          Ritchie, M. E.<i>
et al.</i> limma powers differential expression analyses for RNA-sequencing and
microarray studies. <i>Nucleic acids research</i> <b>43</b>, e47,
doi:10.1093/nar/gkv007 (2015).</span></p>

<p class=EndNoteBibliography0 style='margin-left:36.0pt;text-indent:-36.0pt'><span
lang=EN-US style='font-family:"Arial",sans-serif'>7          Schäfer, J. &amp;
Strimmer, K. A shrinkage approach to large-scale covariance matrix estimation
and implications for functional genomics. <i>Statistical applications in
genetics and molecular biology</i> <b>4</b>, Article32,
doi:10.2202/1544-6115.1175 (2005).</span></p>

<p class=EndNoteBibliography0 style='margin-left:36.0pt;text-indent:-36.0pt'><span
lang=EN-US style='font-family:"Arial",sans-serif'>8          Efron, B. J. J. o.
t. A. S. A. Large-scale simultaneous hypothesis testing: the choice of a null
hypothesis.  <b>99</b>, 96-104 (2004).</span></p>

<p class=a4 style='margin-top:0cm;margin-right:36.75pt;margin-bottom:7.8pt;
margin-left:36.75pt;text-indent:30.0pt'><span lang=EN-US style='font-size:12.0pt;
line-height:150%;color:red'>&nbsp;</span></p>

</div>
    </div>
</div>
</div>


{% endblock %}



